packages/block-library/src/navigation-menu-item/block.json

```diff
@@ -8,6 +8,10 @@
 		"destination": {
 			"type": "string"
 		},
+		"rel": {
```

Comment:
It seems the (block) attribute rel can be computed from the attributes opensInNewTab and nofollow. If that's the case can we remove the rel attribute, and on the save function depending on the attributes opensInNewTab and nofollow we save the current value for the rel (HTML attribute)?

Reply:
yes, but why? Isn't rel an attribute anyway? Is there a reason to have less attributes, especially when we are actually printing attributes on render?

Reply:
The reason is that each time we update the original attribute we also need to update the derived attribute forgetting to do that introduces bugs. In the future, one may easily add a logic that changes nofollow attribute and forgets to change the rel attribute. Another reason is that saving an attribute that can be computed adds more information in the database without a need.